core/src/main/java/com/novoda/noplayer/NoPlayer.java

```diff
@@ -170,9 +186,10 @@
     /**
      * Clear and hide the subtitles on an attached PlayerView.
      *
+     * @return whether the hide was successful.
      * @throws IllegalStateException - if called before {@link NoPlayer#loadVideo(Uri, ContentType)}.
      */
-    void hideSubtitleTrack() throws IllegalStateException;
+    boolean hideSubtitleTrack() throws IllegalStateException;
```

Comment:
Changed this to return `boolean` to be consistent with the other clear methods introduced.